docs/source/markdown/options/userns.container.md

```diff
@@ -22,7 +22,7 @@ Valid _mode_ values are:
 
 **auto**[:_OPTIONS,..._]: automatically create a unique user namespace.
 
-The `--userns=auto` flag, requires that the user name `containers` and a range of subordinate user ids that the Podman container is allowed to use be specified in the /etc/subuid and /etc/subgid files.
++The `--userns=auto` flag requires that the user name `containers` and a range of subordinate user ids, that the Podman container is allowed to use, be specified in the /etc/subuid and /etc/subgid files.
```

Comment:
Is this plus sign intentional?

I still find the wording awkward. The commas don't really help. I think this might be more readable.
>..requires that the user name `containers` be specified in the `/etc/subuid` and `/etc/subgid` files, with a range of subordinate user IDs that the Podman container is allowed to use. See subuid(5).